enforcer.go

```diff
@@ -792,3 +775,22 @@ func (p enforceParameters) Get(name string) (interface{}, error) {
 		return nil, errors.New("No parameter '" + name + "' found.")
 	}
 }
+
+func generateEvalFunction(functions map[string]govaluate.ExpressionFunction, parameters *enforceParameters) govaluate.ExpressionFunction {
+	return func(args ...interface{}) (interface{}, error) {
+		if len(args) != 1 {
+			return nil, fmt.Errorf("Function eval(subrule string) expected %d arguments, but got %d", 1, len(args))
+		}
+
+		expression, ok := args[0].(string)
+		if !ok {
+			return nil, errors.New("Argument of eval(subrule string) must be a string")
+		}
+		expression = util.EscapeAssertion(expression)
+		expr, err := govaluate.NewEvaluableExpressionWithFunctions(expression, functions)
+		if err != nil {
+			return nil, fmt.Errorf("Error while parsing subrule: %s, %s", expression, err.Error())
```

Comment:
I think the subrule is confusing, perhaps use ``eval parameter`` instead of subrule